Cargo.lock

```diff
@@ -3380,14 +3387,20 @@ version = "0.82.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "0559cc0f1779240d6f894933498877ea94f693d84f3ee39c9a9932c6c312bd70"
 
+[[package]]
+name = "wasmparser"
+version = "0.83.0"
```

Comment:
Could wasmparser be updated elsewhere in wasmtime to ensure we've only got one at a time?